cli/ops/io.rs

```diff
@@ -157,7 +157,7 @@ impl StreamResourceHolder {
 pub enum StreamResource {
   Stdin(tokio::io::Stdin, TTYMetadata),
   FsFile(Option<(tokio::fs::File, FileMetadata)>),
-  TcpStream(tokio::net::TcpStream),
+  TcpStream(Option<tokio::net::TcpStream>),
```

Comment:
I don't understand this change

Reply:
As above, Because StreamResourceHolder implements Drop, it cannot be moved